packages/metro-config/src/loadConfig.js

```diff
@@ -46,8 +46,12 @@ function overrideArgument<T>(arg: Array<T> | T): T {
 }
 
 const explorer = cosmiconfig('metro', {
-  searchPlaces: ['metro.config.js', 'metro.config.json', 'package.json'],
-
+  searchPlaces: [
+    'metro.config.js',
+    'metro.config.cjs',
```

Comment:
I think you might have to add `cjs` to `loaders` section too

Reply:
Great spot! That got caught by our internal CI too :)